### ISSUE TITLE
[Core] Module name for sidebar

### ISSUE THREAD

**Issue by armababy** [AsgardCms/Platform#182]
Problem: http://forum.asgardcms.com/topic/37/sidebar-not-showing

It seems that ``$module->getName()`` always return flat module name. Making impossible to use two+ word camel case module names.

**Comment by nWidart** [comment]
Sadly that's related to the module package, not on asgardcms.

**Comment by armababy** [comment]
Yeah was browsing pingpong saw that it returns flat name.

What if we use something like ``$module->get('name')``?
It will return correct one from json object [Module.php](https://github.com/pingpong-labs/modules/blob/f10f5ad2c32f873d98e9669cf471a314d078da09/Module.php#L189-L192)

**Comment by nWidart** [comment]
Interesting, this might be worth trying indeed.

**Comment by armababy** [comment]
Yeah as far as i can test it works consistently, since module will not work without module.json and required attributes we can pretty much rely on this.

vendor/ImageDimensionValue
```
Imagedimensionvalue | studly_case($module->getName()
imagedimensionvalue | $module->getName()
Imagedimensionvalue | $module->getStudlyName()
ImageDimensionValue | $module->get('name')
```
Want pull request or you will do it?

**Comment by n3ISe** [comment]
@armababy thanks for the solution. Double Thumbs Up

**Comment by armababy** [comment]
Damn all this studly_case, hypen, dash, camelcase situation so broken :)
Once you create camelcase module and then try to install it from repository name is getting messed up again, it creates folder with ucfirst it seems.

**Comment by armababy** [comment]
Fixed and merged asgardcms/core#85
